README.md

```diff
@@ -65,12 +65,13 @@ Use the upcoming calendar events below to add the community call events to your
 
 | Date & time | Agenda |
 |-------------|--------|
-| Tuesday Oct 8 <sup>th</sup>, 2024 9:00am Pacific Time (PST)<br />[_See it in your local time and add to your calendar_](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20241008T09&p1=234&msg=Radius+Community+Call) | radius-project/community#49
+| Tuesday November 12 <sup>th</sup>, 2024 9:00am Pacific Time (PST)<br />[_See it in your local time and add to your calendar_](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20241112T09&p1=234&msg=Radius+Community+Call) | radius-project/community#54
 
 ### Previous calls
 
 | Date & time | Agenda |
```

Comment:
should we have just the last 5 calls in the table?

Reply:
yeah that might make it most clean, i'll update it before merging